modules/ROOT/pages/aes-oauth-faq.adoc

```diff
@@ -0,0 +1,27 @@
+= OAuth 2 Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In Anypoint Platform, you can work with OAuth solutions on both the server side, for example, by configuring an HTTP or Salesforce connector to access an OAuth-protected server, and the client side, for example, by applying an OAuth 2.0 policy and and enforce authentication from your client applications through your own OAuth provider.
+
+You can use the following OAuth provider and policy pairs:
+
+[%header,cols="40a,60a"]
+|===
+| Provider | Policy Name
+| PingFederate | PingFederate OAuth Token Enforcement.
+| OpenAM | OpenAM OAuth Token Enforcement.
+| A Dynamic Client Registration-compliant identity provider  | OpenID Connect Access Token Enforcement.
+| Mule OAuth 2.0 | OAuth 2.0 Access Token Enforcement Using External Provider.
```

Comment:
Incorrect description. Please update to Mule OAuth 2.0 Access Token Enforcement.

Reply:
Would it be  like this? https://github.com/mulesoft/docs-api-manager/pull/20/commits/18ea2f2bf2fca15ee993efd6da8a3f73e31b182c

Reply:
Yes. It's just right!